README.md

```diff
@@ -127,7 +127,7 @@ npm run build
 
 # Start server
 ## Note: You need MongoDB running
-npm start
+npm prod
```

Comment:
Thanks a lot for pointing that out! I was wondering why the commands differ in how they work.